echopype/utils/data_proc_lvls.py

```diff
@@ -0,0 +1,66 @@
+"""
+Contains common functions used throughout the data
+processing levels.
+"""
+import pathlib
+from typing import Optional, Tuple, Union
+
+import xarray as xr
+
+from echopype.utils.io import check_file_existence, get_file_format, validate_output_path
+
+
+def validate_source_Sv(
```

Comment:
I just realized that there is no part of this function that requires that the input correspond to `Sv` data. So we can use this in other functions (in particular `apply_mask`) I suggest we remove any mention of `Sv` and replace it with `ds`.

Reply:
I agree with this.

Also I think we should move this to `utils.io` instead of having a separate module for this, especially since the functions used are from `utils.io` already. If there are needs in the future to have many of this type of functions, we can split them out to a separate module under `utils`.